frontend/src/routes/(root)/(logged)/workspace_settings/+page.svelte

```diff
@@ -248,6 +250,12 @@
 				? settings.large_file_storage?.s3_resource_path?.replace('$res:', '')
 				: undefined
 		gitSyncResourcePath = settings.git_sync?.git_repo_resource_path?.replace('$res:', '')
+
+		// check openai_client_credentials_oauth
+		const resourceTypes = await ResourceService.listResourceTypeNames({
```

Comment:
improvement: check for existence of the resource type directly (existsResourceType or getResourceType)